tests/test_checks.py

```diff
@@ -401,15 +401,18 @@ def test_sheets_returns_true_if_neighbors_all_have_values(mocker):
 
 @pytest.mark.google
 def test_sheets_can_find_workspace():
-    patient = GSheetChecker('fake.table', '11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ', 'SGID Stewardship Info')
+    patient = GSheetChecker(
+        'fake.table', '11ASS7LnxgpnD0jN4utzklREgMf1pcvYjcXcIcESHweQ', 'SGID Stewardship Info', 'client-secret.json'
```

Comment:
**question**: Does this depend on there being a `client-secret.json` file in the project? Or is that taken care of in the testing framework?

Reply:
all the tests marked as `google` require a service account json file. It's doc'd in the readme under local development.